src/transaction.cc

```diff
@@ -548,20 +548,54 @@ int Transaction::addRequestHeader(const std::string& key,
 
     if (keyl == "cookie") {
         size_t localOffset = m_variableOffset;
+        size_t pos;
         std::vector<std::string> cookies = utils::string::ssplit(value, ';');
         for (const std::string &c : cookies) {
-            std::vector<std::string> s = utils::string::split(c,
-               '=');
-            if (s.size() > 1) {
-                if (s[0].at(0) == ' ') {
-                    s[0].erase(0, 1);
+            // skip empty substring, eg "Cookie: ;;foo=bar"
+            if (c.length() == 0) {
```

Comment:
c.empty() instead of testing for equality with 0

Reply:
Changed in e14f3dc.